test/tests/shouldRun.test.ts

```diff
@@ -241,6 +241,7 @@ it('should use "type" as a last resort', async () => {
 
   expect(model.schema.path('propy')).toBeInstanceOf(mongoose.Schema.Types.Number);
 
+  // @ts-ignore
```

Comment:
why is an `ts-ignore` here?

Reply:
TypeScript complains that it's being given a number when it expects a string.

Reply:
fixed in 3f5e61e7e35a87c76ceef26cc83517e2f0cd0e68